metadata_ingest/form_parsers.py

```diff
@@ -112,11 +111,11 @@ def generate_dtg_metadata(self):
         default_category = 'Automobiles'
         default_contact_email = 'RDAE_Support@bah.com'
 
-        contactPoint_dataSteward = [i for i in q['contactPoint'] if i['hasRole'] == 'dataSteward'][0]
+        contact_point_data_steward = [i for i in q['contact_points'] if i['hasRole'] == 'dataSteward'][0]
```

Comment:
changes to the dictionary key here should be reverted (`contact_points` is not an expected key in the dictionary)